src/Network/Skylark/Core/Types.hs

```diff
@@ -217,85 +214,83 @@ instance Var LogLevel where
 
 instance (Var a, Show a, Read a) => Var (Maybe a) where
   toVar (Just v) = (unpack . show) v
-  toVar Nothing = ""
+  toVar Nothing  = ""
+
   fromVar = readMaybe
 
 -- | A record type representing full or partial configuration of an
 -- HTTP service. Remaining unspecified fields are filled in with the
 -- default values from Default.
 --
-data Config = Config
-  { _cConfigFile    :: Maybe String   -- ^ Service configuration file location
-  , _cPort          :: Maybe Word32   -- ^ Port to listen on
-  , _cTimeout       :: Maybe Word32   -- ^ Connection timeout (sec)
-  , _cLogLevel      :: Maybe LogLevel -- ^ Logging level
+data Conf = Conf
+  { _confFile     :: Maybe String   -- ^ Service configuration file location
+  , _confPort     :: Maybe Int      -- ^ Port to listen on
```

Comment:
The problem here is that these can be negative. This is why I switched them to Word32s.